src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,64 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int STORAGE_CAPACITY = 10;
+    private int size = 0;
+    private Cell[] cells = new Cell[STORAGE_CAPACITY];
```

Comment:
```suggestion
    private Cell<K, V>[] cells;
```
move the initialization to constructor